APIManager.js

```diff
@@ -0,0 +1,62 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+  constructor() {
+    this.data = {
+      userInfo: {},
+      users: [],
+    };
+  }
+  userData() {
+    $.ajax({
+      method: "GET",
+      url: "https://randomuser.me/api/",
+      dataType: "json",
+      success: (data) => {
```

Comment:
This is kind of repetitive. Try something like:
`const person = data.results[0]`

Then:

`this.data.user = { firstName: person.name.first, lastName: person.name.last, .... }`

It will be less of the same exact words.